cmd/dlv/dlv_test.go

```diff
@@ -156,7 +156,12 @@ func testOutput(t *testing.T, dlvbin, output string, delveCmds []string) (stdout
 
 	_, err = os.Stat(debugbin)
 	if err == nil {
-		t.Errorf("running %q: file %v was not deleted\nstdout is %q, stderr is %q", delveCmds, debugbin, stdout, stderr)
+		if runtime.GOOS != "windows" {
```

Comment:
We should actually just check if we're running in AppVeyor via env vars it sets, we should still run this test otherwise (if people are running tests on their machines, VMs, etc). https://www.appveyor.com/docs/environment-variables/

Also, I wonder what change in AppVeyor led to this suddenly failing.